### ISSUE TITLE
data_range parameter not specified - error

### ISSUE THREAD

**Issue by MyBiscuit** [vaquerizaslab/chess#59]
I was running chess sim on two .mcool maps with this command:
```
chess sim --background-query -p 16 /scratch/ak8725/az_new/az_new.mcool@1000 \
    /scratch/ak8725/oruf_mrg/ruf_mrg.mcool@1000 \
    az_oruf_SBs.bedpe az_oruf_chess_results.tsv
```

Reference and query contact data loaded successfully, but then I received this error:
```
2023-07-16 16:57:05,513 INFO Loading region pairs
2023-07-16 16:57:05,532 INFO Launching workers
2023-07-16 16:57:15,754 INFO Submitting pairs for comparison
2023-07-16 16:57:15,765 ERROR Worker ea63cf5a-97a8-465c-ac2a-da9e109f4d2b encountered a problem: Since image dtype is floating point, you must specify the data_range parameter. Please read the documentation carefully (including the note). It is recommended that you always specify the data_range anyway.
2023-07-16 16:57:15,768 ERROR Worker cf7c8e87-2ad0-4a4a-b69c-6834cf8ab502 encountered a problem: Since image dtype is floating point, you must specify the data_range parameter. Please read the documentation carefully (including the note). It is recommended that you always specify the data_range anyway.
2023-07-16 16:57:15,772 ERROR Worker e3446ca3-ffe0-485d-96fc-f2fc7ff5fe26 encountered a problem: Since image dtype is floating point, you must specify the data_range parameter. Please read the documentation carefully (including the note). It is recommended that you always specify the data_range anyway.
2023-07-16 16:57:15,773 ERROR Worker e19def2d-67c9-432b-8c59-e042659a7014 encountered a problem: Since image dtype is floating point, you must specify the data_range parameter. Please read the documentation carefully (including the note). It is recommended that you always specify the data_range anyway.
Traceback (most recent call last):
  File "/ext3/3.8/bin/chess", line 585, in <module>
    Chess()
  File "/ext3/3.8/bin/chess", line 75, in __init__
    getattr(self, args.command)([sys.argv[0]] + sys.argv[option_ix:])
  File "/ext3/3.8/bin/chess", line 221, in sim
    raise out
ValueError: Since image dtype is floating point, you must specify the data_range parameter. Please read the documentation carefully (including the note). It is recommended that you always specify the data_range anyway.
```
I couldn't find data_range parameter in the chess code, it seems to be a scikit-image parameter. Why am I getting this error and how can I specify the data_range parameter?

P.S. same error occured when I used .hic files.

**Comment by nickmachnik** [comment]
Hey, thanks for raising this issue. This is most l likely a consequence of [this change](https://github.com/scikit-image/scikit-image/pull/6612) in scikit-image. You could technically use a scikit-image release that dates before this change (that would be v0.19.0 or prior).
It seems that this change was made for a good reason though, so we it should be incorporated in chess.
For this we have to provide `data_range` as an argument to `structural_similarity` at every call, for example like this:
```
r_range = reference.max() = reference.min()
q_range = query.max() = query.min()
structural_similarity(reference, query, win_size=window_size, data_range=max(r_range, q_range))
```

**Comment by MyBiscuit** [comment]
How would one incorporate that in the chess sim command?

**Comment by MyBiscuit** [comment]
Alternatively, how can I install chess with the dated scikit-image? I installed chess using 
`pip install chess-hic
`

**Comment by MyBiscuit** [comment]
Thank you.

**Comment by nickmachnik** [comment]
We've published a new release, [3.8.0](https://pypi.org/project/chess-hic/), which you can simply pip install. This should fix the bug you reported.

**Comment by aminakur** [comment]
Successfully installed and ran the new release without any issues, thank you!
